spyder_notebook/notebookplugin.py

```diff
@@ -145,8 +145,12 @@ def get_plugin_actions(self):
                                        _("Save as..."),
                                        icon=ima.icon('filesaveas'),
                                        triggered=self.save_as)
+        open_action = create_action(self,
+                                    _("Open..."),
+                                    icon=ima.icon('fileopen'),
+                                    triggered=self.open_notebook)
         # Plugin actions
-        self.menu_actions = [create_nb_action, save_as_action]
+        self.menu_actions = [create_nb_action, save_as_action, open_action]

```

Comment:
Please change the order of this list to be

    self.menu_actions = [create_nb_action, open_action, save_as_action]

to follow a similar order to the one we have in Spyder's File menu.

Then this will be ready :-)